app/src/test/kotlin/com/intive/picover/images/repository/ImagesRepositoryTest.kt

```diff
@@ -36,15 +36,13 @@ class ImagesRepositoryTest : ShouldSpec(
 			runTest(testDispatcher) {
 				val uriResult: Uri = mockk()
 
-				every { imageReference.listAll() } returns mockk {
-					coEvery { await().items } returns listOf(
-						mockk {
-							every { downloadUrl } returns mockk {
-								coEvery { await() } returns uriResult
-							}
-						},
-					)
-				}
+				coEvery { imageReference.listAll().await().items } returns listOf(
+					mockk {
+						every { downloadUrl } returns mockk {
+							coEvery { await() } returns uriResult
+						}
+					},
+				)
```

Comment:
Please verify once again. There is one redundant mockk. Please also check other tests

Reply:
Updated.

Reply:
What about this? https://github.com/Android-Guild/Picover/pull/315#discussion_r1358134539

Reply:
Let's keep the conversation here. Done

Reply:
What about `PartiesRepositoryTest`?

Reply:
Please suggest which one should be more chained in PartiesRepositoryTest as I don't see any.

Reply:
Please Tag me so I can see it in the notification.

Reply:
@kmaslowiec One important rule which seems to be not clear. If someone places a suggestion, fix or improvement, then you should verify all files in your PR, not only the line where comment was posted.  
1. chained calls seems to be ok
2. please use approach of given-when-then in every file, added in you PR #315, e.g. line 44 has redundant line, please also check rest of the lines in `PartiesRepositoryTest.kt`

Reply:
@MarekMacko Thanks for a detailed explanation. I've checked the files and now both seem to follow given-when-then pattern.

Reply:
Nice! So we're ready to go!